Core/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<modelVersion>4.0.0</modelVersion>
+<groupId>dk.s4_g1</groupId>
+<artifactId>Core</artifactId>
+<packaging>jar</packaging>
+<version>0.0.1</version>
+
+<properties>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <project.java.version>15</project.java.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <maven.compiler.source>${project.java.version}</maven.compiler.source>
+    <maven.compiler.target>${project.java.version}</maven.compiler.target>
+
+</properties>
+
+<dependencies>
+
+    <dependency>
+        <groupId>dk.s4_g1</groupId>
+        <artifactId>Common</artifactId>
+        <version>0.0.1</version>
+    </dependency>
+
+</dependencies>
+
+<!-- Build! -->
+<build>
+    <plugins>
+        <!-- Build -> plugin -> surefire -->
+        <!-- <plugin> -->
+        <!--     <groupId>org.apache.maven.plugins</groupId> -->
+        <!--     <artifactId>maven-surefire-plugin</artifactId> -->
+        <!--     <version>3.0.0-M5</version> -->
+        <!-- </plugin> -->
+        <!-- Build -> plugin -> jacoco -->
+        <!-- <plugin> -->
+        <!--     <groupId>org.jacoco</groupId> -->
+        <!--     <artifactId>jacoco-maven-plugin</artifactId> -->
+        <!--     <version>0.8.6</version> -->
+        <!--     <executions> -->
+        <!--         <execution> -->
+        <!--             <id>coverage-initialize</id> -->
+        <!--             <goals> -->
+        <!--                 <goal>prepare-agent</goal> -->
+        <!--             </goals> -->
+        <!--         </execution> -->
+        <!--         <execution> -->
+        <!--             <id>coverage-report</id> -->
+        <!--             <phase>post-integration-test</phase> -->
+        <!--             <goals> -->
+        <!--                 <goal>report</goal> -->
+        <!--             </goals> -->
+        <!--         </execution> -->
+        <!--     </executions> -->
+        <!-- </plugin> -->
```

Comment:
If we don't need this code now (which we don't since it's commented out) just remove it entirely to prevent zombie-code.